app/src/main/res/layout/activity_add_sale.xml

```diff
@@ -56,16 +51,16 @@
         app:layout_constraintTop_toTopOf="parent" />
 
     <EditText
-        android:id="@+id/filling_ISBN"
-        android:layout_width="257dp"
+        android:id="@+id/fill_in_ISBN"
+        android:layout_width="256dp"
         android:layout_height="48dp"

```

Comment:
Does this work on very little screens (e.g. Cirrus)?

Reply:
It does work fine. I'm not expert enough (plus this is a rather old change) to explain what's going on under the hood though.